### ISSUE TITLE
Black line in place of status bar

### ISSUE THREAD

**Issue by alexsanderkhitev** [eBay/NMessenger#49]
Hello! I pushed ChatMessageViewController (This class of heir NMessengerViewController) I got the black line in place of status bar. When I presented modally ChatMessageViewController I also saw the black line. How can I change it? Thanks 

![bug1](https://cloud.githubusercontent.com/assets/12511909/18646642/852e3074-7ec3-11e6-8c54-9a19fc04c856.png)
![bug2](https://cloud.githubusercontent.com/assets/12511909/18646656/96941342-7ec3-11e6-9339-ac2aabc31612.png)

I push ChatMessageViewController
```
 func didSelectIterlocutor(_ iterlocutor: User) {
        print(iterlocutor.id, iterlocutor.displayName)
        let chatMessengerVC = ChatMessageViewController()
        navigationController?.pushViewController(chatMessengerVC, animated: true)
//        present(chatMessengerVC, animated: true, completion: nil)
    }
```
```

ChatMessageViewController 

import UIKit
import Messenger
import AsyncDisplayKit

class ChatMessageViewController: NMessengerViewController {
    
    // MARK: - Lifecycle

    override func viewDidLoad() {
        super.viewDidLoad()
        // ui 
        showTabBar(bool: false)
        // settings
        setupSettings()
        
    }
    
    // MARK: - Settings 
    
    fileprivate func setupSettings() {
        automaticallyAdjustsScrollViewInsets = false
    }

}

**Comment by alexsanderkhitev** [comment]
I could to resolve this problem adding of whiteView but I think it is bad solution

```
 let whiteView = UIView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: 20))
        whiteView.backgroundColor = .white
        view.addSubview(whiteView)
```

**Comment by DavidSchechter** [comment]
I have a fix for this issue. However, I cannot merge it because of issues with #42. Once those are resolved I will merged a fix for this issue.

**Comment by DavidSchechter** [comment]
Fixed in #53
